generators/python/sdk/features.yml

```diff
@@ -19,6 +19,12 @@ features:
   - id: PAGINATION
     description: |
       Paginated requests will return a `SyncPager` or `AsyncPager`, which can be used as generators for the underlying object.
+  
+  - id: ACCESS_RAW_RESPONSE_DATA
+    advanced: true
+    description: |
+      The SDK provides access to raw response data, including headers, through the `.with_raw_response` property.
+      The `.with_raw_response` property returns a "raw" client that can be used to access the `.headers` and `.data` attributes.
```

Comment:
It might be worth adding a separate section tailored to streaming (for any API that defines a streaming endpoint), given that the UX is different / requires the context manager.

We might be able to use an `id` like `ACCESS_RAW_RESPONSE_DATA_(STREAMING)`, but we'll need to see if the title is rendered as I'd expect. If not, we've meant to add a separate `title` option for a while, so we can do that in a follow-up.

Reply:
I actually wrote the code that generates these snippets to be dynamic. So for streaming endpoints, the generated `README.md` demonstrates an example for the streaming use case.

For example, if you look at `seed/python-sdk/streaming/no-custom-config/README.md`, it shows the following.
```python
from seed import SeedStreaming
client = SeedStreaming(..., )
with client.dummy.with_raw_response.generate_stream(...) as response:
    print(response.headers)  # access the response headers
    for chunk in response.data:
        print(chunk)  # access the underlying object(s)```